OpenSim/Simulation/Model/ContactGeometry.cpp

```diff
@@ -67,11 +67,11 @@ void ContactGeometry::constructProperties()
 {
     constructProperty_location(Vec3(0));
     constructProperty_orientation(Vec3(0));
-    constructProperty_display_preference(1);
+    Appearance defaultAppearance;
+    defaultAppearance.set_color(SimTK::Vec3(0., 1.0, 1.0));
```

Comment:
`SimTK::Cyan` instead? The `SimTK` colors are defined [here](https://github.com/simbody/simbody/blob/master/SimTKcommon/Geometry/src/DecorativeGeometry.cpp#L34).